src/components/fargekategori/bekreft-endre-fargekategori-pa-mange-modal.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';
+import {BodyShort, Button, Heading, List, Modal} from '@navikt/ds-react';
+import fargekategoriIkonMapper from './fargekategori-ikon-mapper';
+import {FargekategoriModell} from '../../model-interfaces';
+import './bekreft-endre-fargekategori-pa-mange-modal.css';
+
+interface Props {
+    valgteBrukereFnrs: string[];
+    valgtFargekategori: FargekategoriModell;
+    onBekreft: () => void;
+    onAvbryt: () => void;
+}
+
+export const BekreftEndreFargekategoriPaMangeModal = ({
+    valgteBrukereFnrs,
+    valgtFargekategori,
+    onBekreft,
+    onAvbryt
+}: Props) => {
+    const [loading, setLoading] = useState(false);
+    const onBekreftClick = () => {
+        onBekreft();
+        setLoading(true);
+    };
+
+    return (
+        <Modal
+            open={true} // Rendring styrt utanfrå
+            className="bekreft-fargekategori-pa-mange-modal"
+            closeOnBackdropClick={true}
+        >
+            <Modal.Header>
+                <Heading size="small" level="1">
+                    Er du sikker på at du vil endre kategori?
+                </Heading>
+            </Modal.Header>
+            <Modal.Body>
+                <BodyShort>
+                    Du har valgt {valgteBrukereFnrs.length} personer og kategorien{' '}
+                    {fargekategoriIkonMapper(valgtFargekategori, 'kategori-som-skal-bekreftes')}
+                </BodyShort>
+                <br />
```

Comment:
Hva med ei lita `spacing` i stedet?
```suggestion
                <BodyShort spacing>
                    Du har valgt {valgteBrukereFnrs.length} personer og kategorien{' '}
                    {fargekategoriIkonMapper(valgtFargekategori, 'kategori-som-skal-bekreftes')}
                </BodyShort>
```